### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by rolandfbk**
I implemented the contact section of the index.html file. I worked on it together with my coding partner. Below is the list of the task we did:

1. Contact section created
2. Contact form implemented
3. Footer containing social media contact implemented

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -185,42 +184,48 @@ <h3>Skills</h3>
...
+                        <h2>Get started</h2>
+                    </div>
+                    <div class="content set">
+                        <p class="description">
+                            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.
+                        </p>
+                        <form class="user-form" action="https://formspree.io/f/xoqrloon" method="post">
+                            <label for="name">
+                                Your name
+                                <input class="name-font" type="text" name="name" id="name" maxlength="30" value="Anastasiia">
```
Comment (AnselemOdims) [inline index.html:200]: - [ ] Kindly make sure this `input` has the `required` attribute as stated in the project requirements
- [ ] Please kindly include a `placeholder` attribute in place of the `value` attribute

File: index.html [hunk 2]
```diff
@@ -185,42 +184,50 @@ <h3>Skills</h3>
...
+                            </label>
+                            <label for="email" id="email_label">
+                                Your email
+                                <input type="email" name="email" id="email" placeholder="Enter your email address" required>
+                            </label>
+                            <textarea id="message" name="message" placeholder="Enter your message here..." required></textarea>
```
Comment (AnselemOdims) [inline index.html:206]: - [ ] Please ensure that the `textarea` input has a `maxlength` property of value `500` as required.

File: index.html [hunk 3]
```diff
@@ -12,7 +12,6 @@
         <div class="wrapper" id="wrapper">
             <header class="header">
```
Comment (AnselemOdims) [inline index.html:13]: - [ ] Please remove the style-agent margin on the `body` element. You can do that by 

```
body {
  margin: 0
}
```

Comment (AnselemOdims) [inline index.html:13]: - [ ] Please remove the style-agent margin on the `body` element. You can do that by 

```
body {
  margin: 0
}
```

File: index.html [hunk 4]
```diff
@@ -185,42 +184,50 @@ <h3>Skills</h3>
...
-                        <li>Portfolio</li>
-                        <li>About</li>
-                        <li>Contact</li>
-                    </ul>
-                </div>
-                <div class="content-bottom">
-                    <img class="f-img" src="img/image_geometry_menu_2.svg" alt="">
+                            <button class="start-button" type="submit">
+                                Start collaboration
+                            </button>
```
Comment (AnselemOdims) [inline index.html:210]: - [ ] Please kindly implement the visual design for the state interactions of the `button`. These states include `pressed`, `hover` as specified in the original design

image

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @rolandfbk, 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 5]
```diff
@@ -185,42 +184,50 @@ <h3>Skills</h3>
...
```
Comment (mmsesay) [inline index.html:210]: - [ ] I think you forgot to check on this button interaction as suggested by the previous reviewer. I noticed you used the wrong hovering colour. It shouldn't be red. 
Please take a look at the original design:

Original Design |
:--------------------:|


Test Demo of your app|
:-----------------------------:|
a1

**Review by mmsesay — CHANGES_REQUESTED** [review]
Hi @rolandfbk & @MaganaAsiati, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by FrederickMih — APPROVED** [review]
# Status : Approved :tada: :tada: :champagne: 

Hi @rolandfbk , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**